values.yaml

```diff
@@ -395,6 +395,14 @@ server:
 
   # Enables a headless service to be used by the Vault Statefulset
   service:
+    # The externalTrafficPolicy can be set to either Cluster or Local
```

Comment:
I think we can be little more succint here. The linked doc contains this information already.

Reply:
I will just keep the options available and leave the reference.